homeassistant/components/media_player/sonos.py

```diff
@@ -199,6 +199,7 @@ def __init__(self, hass, player):
         self.hass = hass
         self.volume_increment = 5
         self._player = player
+        self._name = None
```

Comment:
Yes, the entity_id creation only handles clashes not recognition. :smile: The name should preferably be IDable by a human. Is the UUID that?